Module-2 | Day-3/Callback.js

```diff
@@ -0,0 +1,58 @@
+var fs = require('fs');
+
```

Comment:
use `const` instead of `var` if you do not wish to modify the imported variable.